src/components/layouts/layout-contained/layout-contained.js

```diff
@@ -14,7 +14,9 @@ import React, { memo } from "react";
 
 const LayoutContained = props => {
   return (
-    <div className={`layout-contained ${props.className}`}>
+    <div
```

Comment:
If props.className was missing it returned undefined. (${props.className && props.className) returned undefined as well.

Reply:
what about `props.className || ""` ?

I'm not sure if undefined matters thought. I mean that undefined is a css class that has nothing attached to it so it shouldn't affect anything stylewise

Reply:
`props.className || "" ` is fine so I implemented it. 'undefined' class might not have styles applied on it but it looks as if an error has occurred and is confusing. For sure it in not part on the BEM practices we followed on this project :)